pkg/controllers/user/logging/utils/daemonset.go

```diff
@@ -135,9 +176,9 @@ func newFluentdDaemonset(name, namespace, clusterName string) *v1beta2.DaemonSet
 					Containers: []v1.Container{
 						{
 							Name:            loggingconfig.FluentdHelperName,
-							Image:           v3.ToolsSystemImages.LoggingSystemImages.FluentdHelper,
+							Image:           "micheliac/fluentd-helper:v0.1.4",
```

Comment:
why are we using this image, can it be the same reference way of other tools images

Reply:
yes, must. Will update them after built